src/handlers/dns/DnsStreamHandler.cpp

```diff
@@ -21,7 +21,7 @@
 namespace visor::handler::dns {
 
 DnsStreamHandler::DnsStreamHandler(const std::string &name, InputStream *stream, const Configurable *window_config, StreamHandler *handler)
-    : visor::StreamMetricsHandler<DnsMetricsManager>(name, window_config)
+    : StreamMetricsHandler(name, window_config, _process_dns_groups(window_config))
```

Comment:
It is a trick here. The first thing I need to do is call parent constructor. So, I cannot initialize group bitset ("_g_enabled") before calling this. so I need to call a function that returns a bitset